RELEASE.md

```diff
@@ -0,0 +1,17 @@
+# Releasing a new version of DeltaChat core
+
+For example, to release version 1.116.0 of the core, do the following steps.
+
+1. Resolve all [blocker issues](https://github.com/deltachat/deltachat-core-rust/labels/blocker).
+
+2. Update the changelog: `git cliff -u -t 1.116.0 -p CHANGELOG.md`.
```

Comment:
I personally find it nicer to use the long cmd line args esp. for documentation, since this way, the options are self-explaining:
```suggestion
2. Update the changelog: `git cliff --unreleased --tag 1.116.0 --prepend CHANGELOG.md`.
```

Also, maybe we should add `Have a look at the updated changelog, and manually improve it if necessary.`